Translations/pt_BR.po

```diff
@@ -1003,41 +994,39 @@ msgstr "MÁX"
 
 #: Source/control.cpp:1543
 msgid "Level Up"
-msgstr ""
+msgstr "Subiu de Nível"
 
 #: Source/control.cpp:1688
 msgid "Skill"
 msgstr "Habilidade"
 
 #: Source/control.cpp:1692
-#, fuzzy
-#| msgid "Staff ({:d} charges)"
 msgid "Staff ({:d} charge)"
 msgid_plural "Staff ({:d} charges)"
-msgstr[0] "Cajado ({:d} cargas)"
+msgstr[0] "Cajado ({:d} carga)"
 msgstr[1] "Cajado ({:d} cargas)"
 
 #. TRANSLATORS: Dam refers to damage. UI constrains, keep short please.
 #: Source/control.cpp:1700
 msgid "Mana: {:d}  Dam: {:d} - {:d}"
-msgstr "Mana:{:d}  Dano:{:d} - {:d}"
+msgstr "Mana: {:d}  Dano: {:d} - {:d}"
 
 #. TRANSLATORS: Dam refers to damage. UI constrains, keep short please.
 #: Source/control.cpp:1702
 msgid "Mana: {:d}   Dam: n/a"
-msgstr "Mana: {:d}   Dano: NA"
+msgstr "Mana: {:d}   Dano: n/d"
 
 #. TRANSLATORS: Dam refers to damage. UI constrains, keep short please.
 #: Source/control.cpp:1705
 msgid "Mana: {:d}  Dam: 1/3 tgt hp"
-msgstr "Mana: {:d}  Dano: 1/3 da vida do oponente"
+msgstr "Mana: {:d}  Dano: 1/3 da vida"
```

Comment:
Idk, and the author does not appear to be following this PR

Reply:
@julealgon yeah, agreed. "do alvo" sounds better. "do oponente" might sound like another player (instead of a monster).

Reply:
So what suggestions do I pick

Reply:
@julealgon 
Maybe adding hp is too much (it could be translated to "pontos de vida", some RPG systems translate like this)
@AJenbo 
The best one for me is:
```suggestion
msgstr "Mana: {:d}  Dano: 1/3 da vida do alvo"
```